docs/prometheus-integration.md

```diff
@@ -207,15 +207,15 @@ duration of syncing internal-networkpolicy
 #### Antrea Proxy Metrics
 
 - **antrea_proxy_sync_proxy_rules_duration_seconds:** SyncProxyRules duration
-of AntreaProxy in seconds
+of Antrea Proxy in seconds
```

Comment:
The content are generated automatically, this change will fail the github check `Validate metrics in Prometheus document match running deployment's...`.
You should update the metric description in pkg/agent/proxy/metrics/metric.go and regenerate this file via ./hack/make-metrics-doc.sh and commit the changes